source/mongorestore.txt

```diff
@@ -647,17 +646,17 @@ Options
    Forces :program:`mongorestore` to validate all requests from clients
    upon receipt to ensure that clients never insert invalid documents into
    the database. For objects with a high degree of sub-document nesting,
-   :option:`--objcheck` can have a small impact on performance.
+   :option:`--objcheck <mongorestore --objcheck>` can have a small impact on performance.
 
 
 .. option:: --drop
 
    Before restoring the collections from the dumped backup, drops the
-   collections from the target database. :option:`--drop` does not drop
+   collections from the target database. THIS WORKS FOR ANDREW: :option:`--drop <mongorestore --drop>` does not drop
```

Comment:
Haha, yes I approve. But a thought:

- Old such anchor links: https://docs.mongodb.com/database-tools/mongorestore/#cmdoption-mongorestore-drop
- New: https://docs-mongodbcom-staging.corp.mongodb.com/2ea1122/database-tools/docsworker-xlarge/snooty-setup/mongorestore#std-option-mongorestore.--drop

Minor, but the new version is a bit more convoluted:
- "std" doesn't have any relevance to the user (similar to our conversation on "master" earlier today for the git branch. (Example manual anchor links for a random [param](https://docs.mongodb.com/manual/reference/parameters/#param.ldapUseConnectionPool), [command-line option](https://docs.mongodb.com/manual/reference/program/mongod/#cmdoption-mongod-port), [conf file option](https://docs.mongodb.com/manual/reference/configuration-options/#security.transitionToAuth))
- Having double hypens in the URL itself is just ... weird. What if this link is passed around between colleagues, or posted bare in a powerpoint? The `.--`  there in the anchor link looks like a typo. Considering that we'll use this same format to link to things that have 2, 1, or 0 hyphens, it might be best not to include a parm / option hyphen(s) in its anchor link URL.